x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/transport/SecurityServerTransportInterceptor.java

```diff
@@ -312,7 +312,7 @@ private <T extends TransportResponse> void sendWithCrossClusterAccessHeaders(
                 }
                 final String remoteClusterAlias = remoteClusterCredentials.clusterAlias();
 
-                if (connection.getTransportVersion().before(TRANSPORT_VERSION_ADVANCED_REMOTE_CLUSTER_SECURITY)) {
+                if (connection.getTransportVersion().before(TRANSPORT_VERSION_ADVANCED_REMOTE_CLUSTER_SECURITY_CCR)) {
```

Comment:
Added this small but important adjustment to bump the version barrier to 8.9 for sending/receving cross-cluster requests because the new CCR actions are not understandable by 8.8 nodes.